STEERING-COMMITTEE.md

```diff
@@ -48,6 +48,37 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
```

Comment:
maybe KISS: "Define and enforce branding guidelines "

Reply:
I like the notion of simplicity, how about

1. "Advise the Trademark owner on the definition and enforcement of Trademark usage and branding guidelines."

Reply:
Done.